media-types.md

```diff
@@ -38,25 +38,41 @@ This section shows where the OCI Image Specification is compatible with formats
 
 **Similar/related schema**
 
-- [application/vnd.docker.distribution.manifest.list.v2+json](https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-2.md#manifest-list) - mediaType is different
+- [application/vnd.docker.distribution.manifest.list.v2+json](https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-2.md#manifest-list)
+  - `.mediaType`: only present on Docker
+  - `.annotations`: only present on OCI
+  - `.[]manifests.annotations`: only present on OCI
+  - `.[]manifests.platform`: ARM v6 is represented as `{"architecture":"arm","variant":"v6"}` on OCI, while represented as `{"variant:armv6l"}` on Docker
```

Comment:
This needs to be a mistake in https://github.com/distribution/distribution/pull/1592/files#diff-6b5275ccafa2776e05d590a9e729b0ec6762300317c928d34adc856d9f11714fR108, right? I'm quite sure no implementation has ever supported anything like `armv6l`. Quotes are wrong as well.

Reply:
Improved text. PTAL.

Reply:
Just fix that erroneous example in upstream. No need to copy it around more. Afaics platform was moved to OCI exactly how it was implemented in Docker (with all the os.* weirdnesses). Theoretically, you can use any value in there. OCI spec even makes it clear that it is implementation-defined. The values that OCI recommends for arm are the same values that Docker uses (and used before that docs example was changed to an invalid value).

Reply:
Removed the note about `.[]manifests.platform` and opened PR https://github.com/distribution/distribution/pull/3371 for fixing Docker spec.

Reply:
Updated to remove `StopSignal` and `Labels`

> The fix is to update docs where needed.

Do we need to update all `v1.md`, `v1.1.md`, and `v1.2.md` in https://github.com/moby/moby/tree/v20.10.8/image/spec ? Or maybe we should have `v1.2.1.md`?

Reply:
I don't think we need to increase version. If you want to can look at the publish date for the old ones as I do think at least most of them are very old but just updating `v1.2` is good enough for me.